content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md

```diff
@@ -110,7 +110,7 @@ Now, we will see how the autoscaler reacts to increased load.
 We will start a container, and send an infinite loop of queries to the php-apache service (please run it in a different terminal):
 
 ```shell
-kubectl run -i --tty load-generator --image=busybox /bin/sh
+kubectl run -it --rm --tty load-generator --image=busybox --restart=Never /bin/sh
```

Comment:
```suggestion
kubectl run --generator=run-pod/v1 -i --rm --tty load-generator --image=busybox --restart=Never /bin/sh
```
- `-t` and `--tty` achieve the same thing
- specifying `run-pod/v1` generator for clarity.

Reply:
```suggestion
k run --generator=run-pod/v1 -it --rm load-generator --image=busybox /bin/sh
```
Miss it. Here even shorter variance.